src/Tree/Entity/Repository/NestedTreeRepository.php

```diff
@@ -134,13 +134,21 @@ public function getRootNodes($sortByField = null, $direction = 'asc')
      * Get the Tree path query builder by given $node
      *
      * @param object $node
+     * @param array  $options
```

Comment:
```suggestion
     * @phpstan-param array{includeNode?: bool}  $options
```

Can we please specify these options with array shapes? so PHPStan understands it